src/Components/Components/src/BindMethods.cs

```diff
@@ -69,134 +69,200 @@ public static MulticastDelegate GetEventHandlerValue<T>(Func<T, Task> value)
             return value;
         }
 
+        /// <summary>
+        /// Not intended to be used directly.
+        /// </summary>
+        public static EventCallback GetEventHandlerValue<T>(EventCallback value)
+            where T : UIEventArgs
+        {
+            return value;
+        }
+
+        /// <summary>
+        /// Not intended to be used directly.
+        /// </summary>
+        public static EventCallback<T> GetEventHandlerValue<T>(EventCallback<T> value)
+            where T : UIEventArgs
+        {
+            return value;
+        }
```

Comment:
I'm assuming the above two are for type checking in the Razor compiler.

Reply:
exactly.